Provider/BaseVideoProvider.php

```diff
@@ -194,11 +202,11 @@ protected function getMetadata(MediaInterface $media, $url)
     }
 
     /**
-     * @param \Sonata\MediaBundle\Model\MediaInterface $media
-     * @param string                                   $format
-     * @param array                                    $options
+     * @param MediaInterface $media
+     * @param string         $format
+     * @param array          $options
      *
-     * @return \Imagine\Image\Box
+     * @return Box
      */
     protected function getBoxHelperProperties(MediaInterface $media, $format, $options = array())
```

Comment:
this should be a separate PR, but we should add an ``array``-typehint to ``$options`` IMO